osx/ka-lite-on-pyrun.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Package script for KA-Lite on PyRun
+#
+# Requirements
+# . git
+#
+# Steps
+# . Execute the PyRun installer from https://downloads.egenix.com/python/install-pyrun into a tmp directory.
+# . Clone KA-Lite into a directory
+# . Create a kalite binary in `/usr/local/bin/` to use PyRun's Python instead of the system Python.
+# . Modify `<ka-lite-folder>/python.sh` so it uses PyRun`s Python.
+
+INSTALL_PYRUN="install-pyrun.sh"
+PYRUN_DIR="/tmp/ka-lite-on-pyrun"
```

Comment:
I guess `mktemp` is our friend here: http://unixhelp.ed.ac.uk/CGI/man-cgi?mktemp